### DISCUSSION TITLE
Go back/forth to previous cursor location

### DISCUSSION THREAD

**Discussion by horenso** [helix-editor/helix#6735] [Q&A] [answered]
Hi,
so I searched a bit online, but sorry if the answer is really obvious. I've been using Helix for two weeks now
and I love it, I think I'm already faster than with VSCode or Neovim. So my question is, is there a way to jump
back and forth through to the previously edited positions? I often jump to definition and I wanna go back to the
previous cursor position without thinking where that was.
thx, Jannis

**Comment by mweihs** [comment, marked as answer]
```
C-o     ... jump back
C-i     ... jump forward
C-s     ... save position to jumplist
Space-j ... open jumplist
```
In the jumplist use `C-n` and `C-p` to move up and down.

EDIT: Quick tip, if you want to find out how to do `something` in helix, in normal mode type `Space-?` and then `something` (in your case `jump`).

**Reply by horenso** [reply]
OMG! thank you for that tip. I'm definitely gonna make heavy use of that!
